lib/elixir/test/elixir/kernel/warning_test.exs

```diff
@@ -1768,6 +1768,65 @@ defmodule Kernel.WarningTest do
     purge(TestMod)
   end
 
+  test "deprecate remote nullary zero-arity calls without parens" do
+    assert capture_err(fn ->
+             Code.eval_string("""
+             defmodule TestMod do
+                def foo(), do: nil
+             end
+
+             TestMod.foo
+             """)
+           end) =~
+             "missing parenthesis on call to TestMod.foo/0. parenthesis are always required on function calls without arguments"
+  after
+    purge(TestMod)
+  end
+
+  test "deprecate remote nullary zero-arity calls without parens in a macro" do
```

Comment:
We can remove this test. :) It is not actually exercising a different branch in the codebase.